lib/elixir/lib/registry.ex

```diff
@@ -976,6 +976,55 @@ defmodule Registry do
     end
   end
 
+  @doc """
+  Returns the size of the Registry (ie The number of registered keys).
```

Comment:
It should read:
Returns the size of the registry (ie. the number of registered keys).

Reply:
No use of "ie" or similar. Please use "that is, ..." instead.